edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/stream/LogsStreamRequestHandler.cs

```diff
@@ -60,7 +64,8 @@ Task ProcessLogsFrame(ArraySegment<byte> bytes)
 
         async Task<LogsStreamRequest> ReadLogsStreamingRequest(IClientWebSocket clientWebSocket, CancellationToken cancellationToken)
         {
-            var buf = new byte[1024];
+            // Max size of the request can be 8k
+            var buf = new byte[MaxLogRequestSizeBytes];
```

Comment:
Does this truncate if longer than the max?

Reply:
Yes, we only read up to this size. Ideally should never exceed the max size - the requests currently are < 1k, so 8k seemed enough. If we feel, I can increase to 16k?